src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
If `variableName === keyName` you can use short syntax:
```suggestion
    name,
    wheels,
    version,
```